client/auth/index.js

```diff
@@ -9,10 +9,13 @@ import page from 'page';
 import config from 'config';
 import controller from './controller';
 
-module.exports = function() {
-	if ( config.isEnabled( 'oauth' ) ) {
+export default function() {
```

Comment:
Super minor but while at it shouldn't we also use an arrow function expression here?

```
export default () => {
```